apps/website/src/pages/blogs.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
```

Comment:
please can you fix the linter errors here? you can run it locally with `pnpn lint:ts:fix`